utility/src/main/java/org/oppia/android/util/parser/image/UrlImageParser.kt

```diff
@@ -152,15 +152,18 @@ class UrlImageParser private constructor(
     override fun onResourceReady(resource: T, transition: Transition<in T>?) {
       val drawable = retrieveDrawable(resource)
       htmlContentTextView.post {
-        htmlContentTextView.width { viewWidth ->
+        htmlContentTextView.afterLayout {
           val padding =
             Rect(
               htmlContentTextView.paddingLeft,
               htmlContentTextView.paddingTop,
               htmlContentTextView.paddingRight,
               htmlContentTextView.paddingBottom
             )
-          proxyDrawable.initialize(drawable, computeBounds(context, drawable, viewWidth, padding))
+          proxyDrawable.initialize(
+            drawable,
+            computeBounds(context, drawable, htmlContentTextView.width, padding)
```

Comment:
Does this actually need to change or can we just pass in the width to the callback as we did before?

Reply:
Changed it back to width we get from callback